content/en/docs/concepts/security/rbac-good-practices.md

```diff
@@ -0,0 +1,101 @@
+---
+reviewers:
+title: RBAC Good Practices
+description: >
+  Principles and practices for good RBAC design for cluster operators.
```

Comment:
In this title, consider expanding the abbreviation “RBAC”.

Reply:
expanded

Reply:
That works (I was actually thinking of expanding “Principles and practices for good RBAC design for cluster operators.”).